community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -70,6 +69,13 @@
  */
 public abstract class GraphDatabaseSettings
 {
+    /**
+     * Data block sizes for dynamic array stores.
+     */
+    public static final int DEFAULT_BLOCK_SIZE = 128;
+    public static final int DEFAULT_LABEL_BLOCK_SIZE = 64;
+    public static final int MINIMAL_BLOCK_SIZE = 16;
+
```

Comment:
These constants are not used in this file, and they will be public API if you leave them here.